### PR TITLE
Refactoring to income to rent

### PR DESCRIPTION
**Pull request by elena-vi**

### CODE REVIEW

**Comment by elena-vi** [comment]
[Tech debt income-api: rename hackney:income to rent.](https://trello.com/c/s3SqPGrx/167-tech-debt-income-api-rename-hackneyincome-to-rent)

File: app/controllers/tenancies_controller.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 class TenanciesController < ApplicationController
   def update
-    income_use_case_factory.set_tenancy_paused_status.execute(
+    rent_use_case_factory.set_tenancy_paused_status.execute(
```
Comment (mrosel) [inline app/controllers/tenancies_controller.rb]: It looks like you've just blanket renamed these... But if feels like we've gone too far the other way. These don't directly relate to `rent`

Comment (elena-vi) [reply app/controllers/tenancies_controller.rb]: will rename this to set_case_paused_status

File: app/jobs/application_job.rb [hunk 2]
```diff
@@ -1,6 +1,6 @@
 class ApplicationJob < ActiveJob::Base
-  def income_use_case_factory
-    @income_use_case_factory ||= Hackney::Income::UseCaseFactory.new
+  def rent_use_case_factory
+    @rent_use_case_factory ||= Hackney::Rent::UseCaseFactory.new
   end
```
Comment (elena-vi) [inline app/jobs/application_job.rb]: keep as rent

**Comment by cob16** [comment]
One of the things here is that we do have a rent API in place. There may be duplicate functionality or a clash of naming (i.e the definition of the domain rent could be different) here. This could cause confusion with developers between the two.

**Pull request closed without merging**